tests/pylint/runpylint.py

```diff
@@ -27,6 +27,7 @@ def __init__(self):
             FalsePositive(r"^E1101.*: FedoraGeoIPProvider._refresh: Instance of 'LookupDict' has no 'ok' member"),
             FalsePositive(r"^E1101.*: HostipGeoIPProvider._refresh: Instance of 'LookupDict' has no 'ok' member"),
             FalsePositive(r"^E1101.*: Method 'PropertiesChanged' has no 'connect' member"),
+            FalsePositive(r"^E1101.*: Instance of '[A-Za-z]+PartitioningModule' has no '(storage_handler|request_handler|send_request)' member"),
```

Comment:
These are our class members. Please, disable the warning at the specific places in the tests. It might be an error in other cases.

Reply:
I will fix the commit message, but the rest is correct, I believe. The errors of this kind (no member) come and go: We had code that was "all right", then produced false positives, then no more, then again false positives... with zero change in the code itself and tests always passing. This is just another instance of that.

Reply:
You cannot mark it as a false positive unless we are sure that it is always a false positive. These member are defined only in the `BlivetPartitioningModule` class, other partitioning modules don't have them and will fail if we try to access them. The following code matches your false positive pattern, but it will fail with a traceback:

```
module = AutoPartitioningModule()
handler = module.storage_handler
```

Reply:
Actually, let's do it -  the errors come from as single test that loops over classes, so it's not as horrible as I thought.

Reply:
Ah, sorry, I missed your comment. My point was that such a line of code will be caught by unit tests, because it will not run (quite obviously). Unless the problem is in the code that has no coverage...